### PR TITLE
Add blog link and summary for Ember Times #92

### PR DESCRIPTION
**Pull request by danielleadams**
## What it does
Adds a blurb about my blog post that I posted after Ember Conf. This is a brief blurb about it - I can write a little more if needed!

### CODE REVIEW

File: source/2019-04-05-the-ember-times-issue-92.md [hunk 1]
```diff
@@ -29,10 +29,9 @@ Curious to learn more? Check out her post on Medium: [3 reasons why you should v
 
 ---
 
-## [Section Title in Title Case 🐹](#section-url)
-<change section title emoji>
-<add your name to author list, top and bottom>
-<add blurb and emoji to "SOME-INTRO-HERE">
+## [Reflecting on being a former noob 👶](https://medium.com/@adamzdanielle/reflecting-on-being-a-former-noob-d5e192907ae)
```
Comment (amyrlam) [inline source/2019-04-05-the-ember-times-issue-92.md]:
```suggestion
## [Reflecting on Being a Former Noob 👶](https://medium.com/@adamzdanielle/reflecting-on-being-a-former-noob-d5e192907ae)
```

File: source/2019-04-05-the-ember-times-issue-92.md [hunk 2]
```diff
@@ -29,10 +29,9 @@ Curious to learn more? Check out her post on Medium: [3 reasons why you should v
...
+
+While realizing she's no longer a beginner, [@adamzdanielle](https://twitter.com/adamzdanielle)'s documents her thoughts on what more experienced developers can do to help those newer to the tech industry and working as developers.
```
Comment (amyrlam) [inline source/2019-04-05-the-ember-times-issue-92.md]:
```suggestion
While realizing she's no longer a beginner, [@danielleadams](https://github.com/danielleadams)' documents her thoughts on what more experienced developers can do to help those newer to the tech industry and working as developers. Check out her [blog post](https://medium.com/@adamzdanielle/reflecting-on-being-a-former-noob-d5e192907ae) on the subject, stemming from her experience being a mentor at EmberConf! 
```

**Review by amyrlam — COMMENTED** [review]
thanks so much for the post! 🎉

left a couple suggestions, let us know what you think

**Comment by danielleadams** [comment]
@amyrlam all looks good! Thanks for the review

**Pull request merged**